File: tests/xcalablemp/global-view/shadow-reflect/C/full3.c
#include <stdio.h>

#pragma xmp nodes p(2,2)

#pragma xmp template t(0:9,0:9)
#pragma xmp distribute t(block,block) onto p

double b[10][10];
#pragma xmp align b[i][j] with t(i,j)
#pragma xmp shadow b[0][*]


int main(){
#pragma xmp reflect (b)

#pragma xmp task on p(1,1)
  {
    printf("PASS\n");
  }

}

File: tests/XMP/global-view/shadow-reflect/F/full3.f90
program test
  call sub
end program test

subroutine sub

!$xmp nodes p(2,2)

!$xmp template t(10,10)
!$xmp distribute t(block,block) onto p

  integer b(10,10)
!$xmp align b(i,j) with t(i,j)
!$xmp shadow b(0,*)

!$xmp reflect (b)

!$xmp task on p(1,1)
  write(*,*) 'PASS'
!$xmp end task
  
end subroutine sub
